arch/arm/core/aarch32/prep_c.c

```diff
@@ -42,6 +42,12 @@ Z_GENERIC_SECTION(.vt_pointer_section) __attribute__((used))
 void *_vector_table_pointer;
 #endif
 
+static inline void set_low_vector(void)
+{
+	const int sctlr_low_vectors_bit = 13;
+	write_sctlr(read_sctlr() &~(1 << sctlr_low_vectors_bit));
```

Comment:
Here

`./include/zephyr/arch/arm/aarch32/cortex_a_r/lib_helpers.h:67:MAKE_REG_HELPER(sctlr,         0, 1, 0, 0);`

Reply:
so they added this `lib_helpers.h` while adding ARMv8 support and now we have the functions from both CMSIS and that header ... what a mess.

I will not complain about using the functions from `lib_helpers.h` since it is already in the tree, but please at least try defining the bit field macros properly:

https://github.com/zephyrproject-rtos/zephyr/blob/5710f609c9810f2e63b64c42e80fd7230940a4cd/include/zephyr/arch/arm/aarch32/cortex_a_r/cpu.h#L52-L55

Reply:
I have changed it to use the HIVECS macro that is defined there, which I hadn't seen.